server/graphql/v2/mutation/ActivitySubscriptionsMutations.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express';
+import { GraphQLNonNull } from 'graphql';
+
+import Channels from '../../../constants/channels';
+import models from '../../../models';
+import { checkRemoteUserCanUseAccount } from '../../common/scope-check';
+import { ActivityAndClassesType } from '../enum/ActivityType';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { ActivitySubscriptions } from '../object/ActivitySubscriptions';
+
+const notificationMutations = {
+  toggleEmailNotification: {
+    type: ActivitySubscriptions,
+    description: 'Toggle email notification subscription for requesting logged-in user',
+    args: {
+      type: { type: new GraphQLNonNull(ActivityAndClassesType) },
+      account: {
+        type: AccountReferenceInput,
+        description: 'Scope account which this notification preference is applied to',
+      },
+    },
+    async resolve(_: void, args, req: express.Request) {
+      checkRemoteUserCanUseAccount(req);
+
+      const collective =
```

Comment:
Admin permission check?

Reply:
Not necessary since on email notifications the only affected is the user receiving the email. In this type of activity, the collective is only about the scope of notifications.